internal/runtime/client/client.go

```diff
@@ -336,7 +336,7 @@ func (c *client) CallExtension(ctx context.Context, hook runtimecatalog.Hook, fo
 	}
 
 	log.Info(fmt.Sprintf("Calling extension handler %q", name))
-	var timeoutDuration time.Duration
+	timeoutDuration := runtimehooksv1.DefaultHandlersTimeoutSeconds * time.Second
```

Comment:
This is consistent with what declared in the API

Reply:
fixed line 622
I think we should left the defaulting here even if it is redundant (thinking to clusterctl alpha plan, where we are going to use a "hacky version" if discovery)